content/en/monitors/guide/non_static_thresholds.md

```diff
@@ -0,0 +1,239 @@
+
+### Monitor
+
+Create 3 monitors in total:
+1. [Metric monitor to alert on total number of hits.](#metric-monitor-to-alert-on-the-total-number-of-hits)
+1. [Metric monitor to calculate the error rate.](#metric-monitor-to-calculate-the-error-rate)
+1. [Composite monitor that triggers an alert if the first two monitors are in an ALERT state.](#composite-monitor)
+
+#### Metric monitor to alert on the total number of hits
+
+The first monitor tracks the total number of hits, both successes and failures. This monitor determines whether the error rate should trigger an alert.
+
+{{< tabs >}}
+{{% tab "UI Configuration" %}}
+  {{< img src="monitors/guide/non_static_thresholds/reference_total_hits.png" alt="Metric monitor configuration with formula to calculate total hits" style="width:100%;" >}}
+
+{{% /tab %}}
+
+{{% tab "JSON Example" %}}
+```
+{
+	"name": "Number of hits",
+	"type": "query alert",
+	"query": "sum(last_5m):sum:shopist.checkouts.failed{env:prod} by {region}.as_count() + sum:shopist.checkouts.success{env:prod} by {region}.as_count() > 4000",
+	"message": "There has been more than 4000 hits for this region !",
+	"tags": [],
+	"options": {
+		"thresholds": {
+			"critical": 1000
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": true,
+		"new_group_delay": 60
+	}
+}
+
+```
+{{% /tab %}}
+{{< /tabs >}}
+
+
+#### Metric monitor to calculate the error rate
+
+The second monitor calculates the error rate. Create a query on the number of errors divided by the total number of hits to get the error rate `a / a+b`:
+
+{{< tabs >}}
+{{% tab "UI Configuration" %}}
+  {{< img src="monitors/guide/non_static_thresholds/reference_error_rate.png" alt="Metric monitor configuration with formula to calculate error rate" style="width:100%;" >}}
+{{% /tab %}}
+
+{{% tab "JSON Example" %}}
+```
+{
+	"name": "Error Rate",
+	"type": "query alert",
+	"query": "sum(last_5m):sum:shopist.checkouts.failed{env:prod} by {region}.as_count() / (sum:shopist.checkouts.failed{env:prod} by {region}.as_count() + sum:shopist.checkouts.success{env:prod} by {region}.as_count()) > 0.5",
+	"message": "The error rate is currently {{value}} ! Be careful !",
+	"tags": [],
+	"options": {
+		"thresholds": {
+			"critical": 0.5
+		},
+		"notify_audit": false,
+		"require_full_window": false,
+		"notify_no_data": false,
+		"renotify_interval": 0,
+		"include_tags": true,
+		"new_group_delay": 60
+	}
+}
+
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+#### Composite monitor
+
+The last monitor is a Composite monitor, which sends an alert only if the two preceding monitors are also both in an **ALERT** state. 
+
+{{< img src="monitors/guide/non_static_thresholds/reference_composite_monitor_config.png" alt="Example composite monitor configuration showing boolean logic to alert if both monitors are in ALERT state" style="width:100%;" >}} -->
```

Comment:
I believe the arrow at the end of the line 229 should be removed, no?

Reply:
Yes! Thanks for catching this!